keyboards/skyloong/gk75/q2/info.json

```diff
@@ -0,0 +1,79 @@
+{
+    "manufacturer": "skyloong",
+    "maintainer": "NaturalZh",
+    "bootloader": "stm32duino",
+    "debounce": 10,
+    "build": {
+        "debounce_type": "asym_eager_defer_pk"
+    },
+    "eeprom": {
+        "wear_leveling": {
+            "backing_size": 4096
+        }
+    },
+    "encoder": {
+        "rotary": [
+            {"pin_a": "A1", "pin_b": "A0"},
+            {"pin_a": "A3", "pin_b": "A2", "resolution": 2},
+            {"pin_a": "A4", "pin_b": "B0", "resolution": 2},
+            {"pin_a": "B3", "pin_b": "B1", "resolution": 2}
+        ]
+    },
+    "features": {
+        "bootmagic": true,
+        "command": true,
+        "encoder": true,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true,
+        "rgb_matrix": true
+    },
+    "indicators": {
+        "on_state": 0,
+        "caps_lock": "C15"
+    },
+
+    "matrix_pins": {
+        "cols": [null, null, null, null, null, null, null, null, null, null, null, null, null],
+        "rows": ["A8", "A9", "A10", "A15", "B6", "B7","B9"]
```

Comment:
Please move these back to `config.h`. This should not be used for custom matrix.

Reply:
I modified  it. Would you please review it again? @fauxpark